docs/BR.md

```diff
@@ -1935,7 +1956,22 @@ The CA SHALL protect its Private Key in a system or device that has been validat
 
 ### 6.3.2 Certificate operational periods and key pair usage periods
 
-Subscriber Certificates issued on or after 1 September 2020 SHOULD NOT have a Validity Period greater than 397 days and MUST NOT have a Validity Period greater than 398 days. 
+Subscriber Certificates issued on or after 1 September 2020 and before 15 September 2025 SHOULD NOT have a Validity Period greater than 397 days and MUST NOT have a Validity Period greater than 398 days.
+
+Subscriber Certificates issued on or after 15 September 2025 and before 15 September 2026 SHOULD NOT have a Validity Period greater than 199 days and MUST NOT have a Validity Period greater than 200 days.
+
+Subscriber Certificates issued on or after 15 September 2026 and before 15 April 2027 SHOULD NOT have a Validity Period greater than 99 days and MUST NOT have a Validity Period greater than 100 days.
+
+Subscriber Certificates issued on or after 15 April 2027 SHOULD NOT have a Validity Period greater than 44 days and MUST NOT have a Validity Period greater than 45 days.
+
+Table: Reference for maximum Validity Periods of Subscriber Certificates
+
+| __Certificate issued on or after__ | __Certificate issued before__ | __Maximum number of days for data reuse__ |
```

Comment:
@clintwilson  I presume you mean the below:

```suggestion
| __Certificate issued on or after__ | __Certificate issued before__ | __Maximum number of days for validity__ |
```

Reply:
Yup! I even have this fixed locally, just forgot to push the commit 🤦‍♀️